WooCommerce/Classes/POS/Models/POSProduct.swift

```diff
@@ -8,12 +10,17 @@ public struct POSProduct {
     // The WooCommerce core API for Product makes stockQuantity Int or null, however some extensions allow decimal values as well.
     // We might want to use Decimal type for consistency with the rest of the app
     public let stockQuantity: Int
+    public let priceWithCurrency: String
+    public let currencySettings: CurrencySettings
 
-    public init(itemID: UUID, productID: Int64, name: String, price: String, stockQuantity: Int) {
+    public init(itemID: UUID, productID: Int64, name: String, price: String, currencySettings: CurrencySettings, stockQuantity: Int) {
```

Comment:
super nit: the dependency could be after the entity fields

```suggestion
    public init(itemID: UUID, productID: Int64, name: String, price: String, stockQuantity: Int, currencySettings: CurrencySettings) {
```